build/Rulesets/NonShippingProjectRules.ruleset

```diff
@@ -15,5 +15,6 @@
     <!-- See above comment under CSharp -->
     <Rule Id="RS0003" Action="None" />
     <Rule Id="RS0007" Action="None" />
+    <Rule Id="RS0005" Action="None" />
```

Comment:
The "Do not use generic CodeAction.Create to create CodeAction" analyzer is crashing when it analyzed BasicEditorServicesTest.  The analyzer isn't of any value to the test code*, and we need to unblock this RI issue, so i'm just disabling it from running here.

* Then analyzer is about not calling CodeAction.Create directly (and instead creating a subclass of DocumentCodeAction).  That not relevant in test code.